mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -140,7 +140,11 @@ class CrossValidator(override val uid: String) extends Estimator[CrossValidatorM
 @Experimental
 class CrossValidatorModel private[ml] (
     override val uid: String,
+
+    /** The best model selected from cross validation */
```

Comment:
@jkbradley Thank you for your clarification. I modified the PR according to your comment. Please have a look. Note that I also rebase to the master branch, since it has been moved.